packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -68,11 +69,16 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
    * @private
    */
   async _applyGenesis(commit: any, meta: CommitMeta, context: Context): Promise<StreamState> {
-    let payload = commit
-    const isSigned = StreamUtils.isSignedCommit(commit)
+    let payload = StreamUtils.isExpandedCommit(commit) ? commit.commit : commit
+    const isSigned = StreamUtils.isSignedCommit(commit) || (commit as CommitData).envelope
```

Comment:
if `StreamUtils.isSignedCommit` is no longer sufficient to properly answer this question then we should update it.  Shouldn't have to do an extra check after calling the function whose sole purpose is answering exactly this question